src/core/public/chrome/ui/header/header_breadcrumbs.scss

```diff
@@ -0,0 +1,55 @@
+@import "../../../../public/variables";
+
+$firstBreadcrumbPolygon: polygon(
+  0% 0%,
+  100% 0%,
+  calc(100% - #{$euiSizeS}) 100%,
+  0% 100%
+);
+$breadcrumbPolygon: polygon(
+  $euiSizeS 0%,
+  100% 0%,
+  calc(100% - #{$euiSizeS}) 100%,
+  0% 100%
+);
+
+.osdHeaderBreadcrumbs {
+  /*
+  filter defines a custom filter effect by grouping atomic filter primitives!
+  here we are using Gaussian filter with stdDeviation for applying
+  border-radius on clipped background.
+  */
+
+  filter: url("../../public/assets/round_filter.svg#round");
+
+  .osdBreadcrumbs {
+    clip-path: $breadcrumbPolygon;
+    background-color: $osdHeaderBreadcrumbGrayBackground;
+    padding: $euiSizeXS - 2.5 $euiSizeL - $euiSizeXS;
```

Comment:
`- 2.5 $euiSizeL` ?

Reply:
padding: 
top and bottom paddings are $euiSizeXS - 2.5   => 1.5px
right and left paddings are $euiSizeL - $euiSizeXS  => 20px